posts/flock_2024_diary/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Packit Flock 2024 Diary"
+date: 2024-09-05T15:16:18+02:00
+authors: flachman
+tags:
+  - fedora
+  - flock
+  - conference
+  - "2024"
+---
+
+# Flock Packit Diary Blog post
+
+It’s been a month since two Packit members (Laura and František) attended Flock, the annual Fedora conference, in Rochester. Let’s see some highlights from our journey There and Back Again. Sadly the travel there wasn’t as expected at all, but luckily the tornadoes did not stop us and we both made it to Rochester.
+
+# Packit contributions
+
+We prepared two talks for the Flock attendees. If you missed it, the recordings are available. But here’s what we presented:
+
+## Packit release automation journey ([recording](https://www.youtube.com/watch?v=x-o91dkv1Cg))
+
+This one mainly targeted the Fedora maintainers and described how you can use Packit for your packages. The basic workflow has been here for some time but we’ve presented a couple of interesting news about what we are currently working on:
+
+The requirement to have a git repository representing is now gone. By NOT specifying the `upstream_project_url` config option, Packit will not try to clone or interact with the upstream via git. This is especially useful if the upstream project does not use git, or you just don’t need it (e.g. packages using language-specific repositories like PyPI).
+
+If you are interested in having a nice git tree, you’ll welcome that the Packit team is currently working on making it possible t[o avoid diverging branches when doing release syncing](https://github.com/packit/packit-service/pull/2478).
+
+Another long-time-awaited feature is support for releasing package groups using side-tags. Since Flock, Nikola from the Packit team polished the functionality and even created a [dedicated documentation pag](https://packit.dev/docs/fedora-releases-guide/releasing-multiple-packages)e describing how to use it.
```

Comment:
```suggestion
Another long-time-awaited feature is support for releasing package groups using side-tags. Since Flock, Nikola from the Packit team polished the functionality and even created a [dedicated documentation page](https://packit.dev/docs/fedora-releases-guide/releasing-multiple-packages)e describing how to use it.
```